rotkehlchen/chain/ethereum/modules/compound/decoder.py

```diff
@@ -207,6 +216,78 @@ def _decode_borrow_and_repay(
+            all_logs: list['EvmTxReceiptLog'],
+    ) -> DecodingOutput:
+        """Decode a liquidation event happening over a tracked account"""
+        borrower = hex_or_bytes_to_address(tx_log.data[32:64])
+        if self.base.is_tracked(borrower) is False:
+            return DEFAULT_DECODING_OUTPUT
+
+        liquidator_address = hex_or_bytes_to_address(tx_log.data[0:32])
+        repay_amount_raw = hex_or_bytes_to_int(tx_log.data[64:96])
+        collateral_ctoken_address = hex_or_bytes_to_address(tx_log.data[96:128])
+        seize_amount_raw = hex_or_bytes_to_int(tx_log.data[128:160])
+
+        collateral_ctoken = get_or_create_evm_token(
+            userdb=self.base.database,
+            evm_address=collateral_ctoken_address,
+            chain_id=self.base.evm_inquirer.chain_id,
+            token_kind=EvmTokenKind.ERC20,
+            evm_inquirer=self.base.evm_inquirer,
+            protocol=CPT_COMPOUND,
+        )
+        seized_collateral_amount = asset_normalized_value(
+            amount=seize_amount_raw,
+            asset=collateral_ctoken,
+        )
+        # use the logs to know what token was getting repayed for the position. We have the ctoken
+        # from the event log but not the underlying token
+        repaying_token = None
+        for event_log in all_logs:
+            if (
+                event_log.topics[0] == ERC20_OR_ERC721_TRANSFER and
+                event_log.topics[1] == tx_log.data[0:32] and
+                hex_or_bytes_to_address(event_log.topics[2]) == tx_log.address
+            ):
+                repaying_token_address = event_log.address
+                repaying_token = get_or_create_evm_token(
+                    userdb=self.base.database,
+                    evm_address=repaying_token_address,
+                    chain_id=self.base.evm_inquirer.chain_id,
+                    token_kind=EvmTokenKind.ERC20,
+                    evm_inquirer=self.base.evm_inquirer,
+                )
+                break
+
+        if repaying_token is None:
+            log.error(f'Failed to decode compound liquidation at {transaction_hash.hex()}')
+            return DEFAULT_DECODING_OUTPUT
+
+        repayed_amount = asset_normalized_value(
+            amount=repay_amount_raw,
+            asset=repaying_token,
+        )
+        for event in decoded_events:
+            if (
+                event.event_type == HistoryEventType.SPEND and
+                event.event_subtype == HistoryEventSubType.NONE and
+                event.location_label == borrower and
+                event.address == liquidator_address and
+                event.balance.amount == seized_collateral_amount and
+                event.asset == collateral_ctoken
+            ):
+                event.event_subtype = HistoryEventSubType.LIQUIDATE
+                event.notes = f'Liquidate {seized_collateral_amount} {collateral_ctoken.symbol} to repay {repayed_amount} {repaying_token.symbol} on compound'  # noqa: E501
```

Comment:
This is from the view of the borrower/victim. So it should be:

Lost {
```suggestion
                event.notes = f'Lost {seized_collateral_amount} {collateral_ctoken.symbol} in a compound forced liquidation to repay {repayed_amount} {repaying_token.symbol}'  # noqa: E501
```